files/jira.rb

```diff
@@ -91,6 +91,19 @@ def project
   def handle
     return false if !should_ticket?
     return false if !project
+
+    1.upto(5).each do |i|
```

Comment:
Instead of this can you reuse the `timeout_and_retry` function? (already used by hipchat and pd)